py4j-java/src/main/java/py4j/GatewayServer.java

```diff
@@ -754,7 +783,7 @@ protected void startSocket() throws Py4JNetworkException {
 	 * @param interfacesToImplement
 	 * @return
 	 */
-	public Object getPythonServerEntryPoint(Class[] interfacesToImplement) {
+	public Object getPythonServerEntryPoint(@SuppressWarnings("rawtypes") Class[] interfacesToImplement) {
```

Comment:
Hi Barthelemy,

It says since 1.5 for the @SuppressWarnings annotation here:

https://docs.oracle.com/javase/7/docs/api/java/lang/SuppressWarnings.html

Although I admittedly haven't tested against either 1.6 or 1.7.